src/clients/python/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[project]
+name = "tigerbeetle_python"
+version = "0.0.1"
+description = "The TigerBeetle client for Python."
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Topic :: Database :: Front-Ends",
+    "Development Status :: 5 - Production/Stable"
+]
+
+[project.urls]
+Homepage = "https://github.com/tigerbeetle/tigerbeetle"
+Issues = "https://github.com/tigerbeetle/tigerbeetle/issues"
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.sdist]
+only-include = ["tigerbeetle_python"]
+artifacts = ["*.so", "*.dylib", "*.dll"]
+
+[tool.hatch.build.targets.sdist.force-include]
```

Comment:
This is used to convert our directory structure (README.md in root) to what Python wants for packagaing (README.md inside package).